foreman/data_refinery_foreman/foreman/management/commands/test_rerun_salmon_old_samples.py

```diff
@@ -0,0 +1,288 @@
+import datetime
+import json
+import os
+
+from django.test import TestCase
+from django.utils import timezone
+from typing import Dict, List
+from unittest.mock import MagicMock, Mock, patch, call
+
+from data_refinery_common.job_lookup import ProcessorPipeline, Downloaders
+from data_refinery_common.models import (
+    ComputationalResult,
+    ComputationalResultAnnotation,
+    ComputedFile,
+    DownloaderJob,
+    DownloaderJobOriginalFileAssociation,
+    Experiment,
+    ExperimentOrganismAssociation,
+    ExperimentResultAssociation,
+    ExperimentSampleAssociation,
+    OriginalFile,
+    OriginalFileSampleAssociation,
+    Organism,
+    OrganismIndex,
+    Processor,
+    ProcessorJob,
+    ProcessorJobOriginalFileAssociation,
+    Sample,
+    SampleResultAssociation,
+)
+from data_refinery_foreman.foreman.management.commands.rerun_salmon_old_samples import update_salmon_all_experiments
+
+def setup_experiment(complete_accessions: List[str], incomplete_accessions: List[str]) -> Dict:
```

Comment:
Can you change the names of the parameters here to `new_version_accessions` and `old_version_accessions` or something like that?